eng/dotnet-format/dotnet-format-integration.yml

```diff
@@ -0,0 +1,92 @@
+jobs:
```

Comment:
Could I get some help about where should I put these in the CI?
It's a couple of tests (run dotnet-format on itself, and other real repo) but I can't figure out what's the correct place to reference this.